src/js/expanderExpanded.js

```diff
@@ -23,6 +23,7 @@ function mdExpanderExpandedDirective($mdUtil, $animateCss) {
 
   function link(scope, element, attr, expanderCtrl) {
     var isHeightSet = expanderCtrl.height !== undefined;
+    var isAnimatedOpen = false;
```

Comment:
Let's add tests for this. If it's a simple ui component, then the tests should be easy to write (relatively)